crates/holochain/src/core/workflow/sys_validation_workflow/validation_deps.rs

```diff
@@ -202,10 +315,63 @@ impl ValidationDependencyState {
         }
     }
 
+    pub(super) fn set_pending_validation_warranted(
```

Comment:
I think docstrings would be nice for these new methods.

Reply:
https://github.com/holochain/holochain/pull/5249/commits/9801bda0488c75727acee3e09527cb747eeaf3d9